config/environments/development.rb

```diff
@@ -27,10 +27,10 @@
     config.cache_store = :null_store
   end
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-
-  config.action_mailer.perform_caching = false
+  config.action_mailer.delivery_method = :notify
```

Comment:
A reminder that when we deployed to edge we saw that we needed to have this config for the different environments.

Reply:
Good catch!

Reply:
Sorted that now 👍

Reply:
I've set it in application.rb, rather than in every single environment, so it cascades down to all environments and gets overridden in test